FWCore/Framework/test/stubs/ToyIntProducers.cc

```diff
@@ -486,6 +486,100 @@ namespace edmtest {
     }
   };
 
+  //--------------------------------------------------------------------
+
+  class NonEventIntProducer : public edm::global::EDProducer<edm::Accumulator,
```

Comment:
Would it be worth of the effort to test also stream, one, and limited?

Reply:
The internals of the framework treat all kinds of modules identically when it comes to launching work at non-event transitions.